http://bugzilla.gnome.org/show_bug.cgi?id=350962

From: Federico Mena Quintero <federico@gnu.org>

https://bugzilla.novell.com/show_bug.cgi?id=117333

Use "gnome-fs-directory" instead of "file-manager" as an icon.  A
folder is more recognizable than a filing cabinet.
---

 nautilus-file-management-properties.desktop.in.in |    2 +-
 nautilus.desktop.in.in                            |    2 +-
 src/nautilus-file-management-properties.c         |    2 +-
 src/nautilus-navigation-window.c                  |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)


Index: nautilus-2.27.1/data/nautilus-file-management-properties.desktop.in.in
===================================================================
--- nautilus-2.27.1.orig/data/nautilus-file-management-properties.desktop.in.in
+++ nautilus-2.27.1/data/nautilus-file-management-properties.desktop.in.in
@@ -3,7 +3,7 @@ Encoding=UTF-8
 _Name=File Management
 _Comment=Change the behaviour and appearance of file manager windows
 Exec=nautilus-file-management-properties
-Icon=system-file-manager
+Icon=gnome-fs-directory
 Terminal=false
 Type=Application
 StartupNotify=true
Index: nautilus-2.27.1/data/nautilus.desktop.in.in
===================================================================
--- nautilus-2.27.1.orig/data/nautilus.desktop.in.in
+++ nautilus-2.27.1/data/nautilus.desktop.in.in
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 _Name=File Manager
 Exec=nautilus
-Icon=system-file-manager
+Icon=gnome-fs-directory
 Terminal=false
 Type=Application
 StartupNotify=true
Index: nautilus-2.27.1/src/nautilus-file-management-properties.c
===================================================================
--- nautilus-2.27.1.orig/src/nautilus-file-management-properties.c
+++ nautilus-2.27.1/src/nautilus-file-management-properties.c
@@ -807,7 +807,7 @@ nautilus_file_management_properties_dial
 			       (GClosureNotify)g_object_unref,
 			       0);
 
-	gtk_window_set_icon_name (GTK_WINDOW (dialog), "system-file-manager");
+	gtk_window_set_icon_name (GTK_WINDOW (dialog), "gnome-fs-directory");
 
 	if (window) {
 		gtk_window_set_screen (GTK_WINDOW (dialog), gtk_window_get_screen(window));
Index: nautilus-2.27.1/src/nautilus-navigation-window.c
===================================================================
--- nautilus-2.27.1.orig/src/nautilus-navigation-window.c
+++ nautilus-2.27.1/src/nautilus-navigation-window.c
@@ -1285,7 +1285,7 @@ static NautilusIconInfo *
 real_get_icon (NautilusWindow *window,
 	       NautilusWindowSlot *slot)
 {
-	return nautilus_icon_info_lookup_from_name ("system-file-manager", 48);
+	return nautilus_icon_info_lookup_from_name ("gnome-fs-directory", 48);
 }
 
 static void
